DataGenerator/DataGenerator.py

```diff
@@ -20,16 +20,16 @@
 import xnat
 
 class DataGenerator(torch.utils.data.Dataset):
-    def __init__(self, mastersheet, label, config, keys, inference=False, n_norm = None, c_norm = None, transform=None, target_transform = None):
+    def __init__(self, mastersheet, label, config, keys, inference=False, transform=None, target_transform = None, **kwargs):
         super().__init__()
         self.keys             = list(keys)
         self.transform        = transform
         self.target_transform = target_transform
         self.label            = label
         self.inference        = inference
         self.mastersheet      = mastersheet
-        self.n_norm = n_norm
-        self.c_norm = c_norm
+        self.n_norm = kwargs['numerical_norm']
```

Comment:
You don't need to pass to the kwargs. The kwargs is only for the function above, in which you don't want to pass a list of all arguments. Buthere you can name them (like was done in version before)